Crates.io build fails on stable-x86_64-pc-windows-msvc

Full rustc version: stable-x86_64-pc-windows-msvc unchanged - rustc 1.19.0 (0ade33941 2017-07-17)

Build error:
```
   Compiling sys-info v0.5.1
error[E0432]: unresolved import `libc::sysctl`
  --> C:\Users\Layl\.cargo\registry\src\github.com-1ecc6299db9ec823\sys-info-0.5.1\lib.rs:16:5
   |
16 | use libc::sysctl;
   |     ^^^^^^^^^^^^ no `sysctl` in the root

error[E0412]: cannot find type `suseconds_t` in module `libc`
   --> C:\Users\Layl\.cargo\registry\src\github.com-1ecc6299db9ec823\sys-info-0.5.1\lib.rs:312:39
    |
312 |         bt.tv_usec = secs[1] as libc::suseconds_t;
    |                                       ^^^^^^^^^^^ not found in `libc`

error[E0308]: mismatched types
   --> C:\Users\Layl\.cargo\registry\src\github.com-1ecc6299db9ec823\sys-info-0.5.1\lib.rs:311:21
    |
311 |         bt.tv_sec = secs[0]  as libc::time_t;
    |                     ^^^^^^^^^^^^^^^^^^^^^^^^ expected i32, found i64

error: aborting due to previous error(s)

error: Could not compile `sys-info`.

To learn more, run the command again with --verbose.
```

--- a/lib.rs
+++ b/lib.rs
@@ -13,6 +13,7 @@ use std::io::{self, Read};
 use std::fs::File;
 use std::os::raw::c_char;
 
+#[cfg(not(windows))]
 use libc::sysctl;
 use libc::timeval;
 use std::mem::size_of_val;
@@ -295,6 +296,7 @@ pub fn hostname() -> Result<String, Error> {
 }
 
 /// Get system boottime
+#[cfg(not(windows))]
 pub fn boottime() -> Result<timeval, Error> {
     let mut bt = timeval {
         tv_sec: 0,

